charts/capsule-proxy/README.md

```diff
@@ -54,7 +54,7 @@ The values in your overrides file `myvalues.yaml` will override their counterpar
 
 If you only need to make minor customizations, you can specify them on the command line by using the `--set` option. For example:
 
-        $ helm install capsule-proxy clastix/capsule-proxy --values myvalues.yaml -n capsule-system
+        $ helm install capsule-proxy clastix/capsule-proxy --set "kind=DaemonSet" -n capsule-system
```

Comment:
Why defining the `kind` value?

Default is `Deployment` but I'm missing the point on why we should document to set the deployment kind to `DaemonSet`.

Reply:
Just to provide an example about `--set` option ;
feel free to swap it with another helm chart property (listening port?).

Reply:
We don't have to provide this kind of example since we're considering operators know how to use Helm.

Please, amend the commit dropping this addition.

Reply:
Ah, my bad: it's good enough! :)